index.html

```diff
@@ -21,10 +21,12 @@
 
     const ML_FUNC = "ML_FUNC";
     const ML_STRUCT = "ML_STRUCT";
+    const ML_QUOTE = "ML_QUOTE";
 
     // var multilineModeEnabled = false;
     const multilineFuncGutter = "\n...\t\t";
     const multilineStructGutter = "\n... ";
+    const multilineQuoteGutter = "\n";
```

Comment:
Use the same gutter as the `multilineStructGutter`. I want it to look like `bpython`:

```
>>> """
... sup
... dude
... """
```

Reply:
ok https://github.com/daniel-schroeder-dev/cwhq-python-repl-hacking/pull/1/commits/57db4946877aea6dda829c55488f176b67edabfd